README.md

```diff
@@ -253,12 +261,27 @@ This .github folder contains workflows and templates related to github
 
 Workflows are in the .github/workflows folder
 
+- `ci.yml` Workflow run when code merged to main. Deploys to dev and qa environments.
 - `combine-dependabot-prs.yml`: Workflow for combining dependabot pull requests. Runs on demand
 - `delete_old_cloudformation_stacks.yml`: Workflow for deleting old cloud formation stacks. Runs daily
 - `pull_request.yml`: Called when pull request is opened or updated. Calls sam_package_code and sam_release_code to build and deploy the code. Deploys to dev AWS account. The main and sandbox stacks deployed have PR-<PULL_REQUEST_ID> in the name
 - `quality_checks.yml`: Runs check-licenses, lint, test and sonarcloud scan against the repo. Called from pull_request.yml and release.yml
-- `release.yml`: Run when code is merged to main branch or a tag starting v is pushed. Calls sam_package_code and sam_release_code to build and deploy the code.
+- `release.yml`: Runs on demand to create a release and deploy to all environments.
 - `sam_package_code.yml`: Packages code and uploads to a github artifact for later deployment
 - `sam_release_code.yml`: Release code built by sam_package_code.yml to an environment
 - `pr-link.yaml`: This workflow template links Pull Requests to Jira tickets and runs when a pull request is opened.
 - `dependabot.yml`: Dependabot definition file
+
+### Github pages
+
+Github pages is used to display deployment information. The source for github pages is in the gh-pages branch.
+As part of the ci and release workflows, the release tag (either the short commit SHA or release tag) is appended to _data/{environment}_deployments.csv so we have a history of releases and replaced in _data/{environment}_latest.csv so we now what the latest released version is.
```

Comment:
The underscores and brackets won't render properly, you probably want the filenames to be in backticks!

Reply:
Nice on. Added in 0ba114fe0eef5ed8166d9f866aaf90a1604977c2